rs/bitcoin/ckbtc/minter/src/test_fixtures.rs

```diff
@@ -142,3 +142,309 @@ pub mod mock {
         }
     }
 }
+
+pub mod arbitrary {
+    use crate::{
+        address::BitcoinAddress,
+        signature::EncodedSignature,
+        state::{
+            eventlog::{Event, EventType},
+            ChangeOutput, Mode, ReimbursementReason, RetrieveBtcRequest, SuspendedReason,
+        },
+        tx,
+        tx::{SignedInput, TxOut, UnsignedInput},
+    };
+    use candid::Principal;
+    pub use event::event_type;
+    use ic_base_types::CanisterId;
+    use ic_btc_interface::{OutPoint, Satoshi, Txid, Utxo};
+    use icrc_ledger_types::icrc1::account::Account;
+    use proptest::{
+        array::uniform20,
+        array::uniform32,
+        collection::{vec as pvec, SizeRange},
+        option,
+        prelude::{any, Just, Strategy},
+        prop_oneof,
+    };
+    use serde_bytes::ByteBuf;
+
+    // Macro to simplify writing strategies that generate structs.
+    macro_rules! prop_struct {
+        ($struct_path:path { $($field_name:ident: $strategy:expr),* $(,)? }) => {
+            #[allow(unused_parens)]
```

Comment:
why is this needed? Is there any way we can remove `unused_parens`?

Reply:
Good question. It seems that this is a bug in the Rust compiler, see e.g. [this discussion](https://stackoverflow.com/questions/74983561/unnecessary-parens-warning-in-macro-call-that-needs-the-parens-is-this-a-bad) and the [linked GitHub issue](https://github.com/rust-lang/rust/issues/73068).